Source/WPE/Headers/WPE/Input/Events.h

```diff
@@ -29,6 +29,11 @@
 
 #include <array>
 
+namespace IPC {
```

Comment:
Same here, this IPC namespace and all the additional changes to this file are not used and shouldn't be necessary.